pages/learn/deploy/release-strategy/update-strategies.md

```diff
@@ -22,6 +22,8 @@ Setting `RESIN_SUPERVISOR_UPDATE_STRATEGY` to a valid value selects the update s
 
 which are explained below. `RESIN_SUPERVISOR_HANDOVER_TIMEOUT` is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
 
+All update strategies below honor the [application update locks](/learn/deploy/release-strategy/update-locking/) which you can use prevent updates temporarily.
```

Comment:
nit: Why the newline though?
nit: Use reference style links where possible. There is no wrong way of doing markdown but trying to achieve consistency all over. Check https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs
Also, change `[Fleet Configuration environment variables](/learn/manage/configuration/)` if you can.

Otherwise, ship it.

Reply:
The current paragraph is focused on supervisor configuration, whereas the addition is not. Therfore, I think a new line makes sense.

Good point about the link-type. I'll fix it